libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -112,6 +112,9 @@ class AP_MotorsHeli_RSC {
 
     // set the throttle percentage to be sent to external governor to signal that autorotation bailout ramp should be used within this instance of Heli_RSC
     void        set_ext_gov_arot_bail(int16_t pct) { _rsc_arot_bailout_pct = pct; }
+	
+    // turbine start initialize sequence
+    void        set_turbine_start(bool turbine_start) {_turbine_start = (bool)turbine_start; }
```

Comment:
I don't think there's any need to cast to a bool because the argument is already a boolean.

Reply:
got it.  Thanks!